ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,37 @@
+Make sure you are using the **latest** version of youtube-dl. Latest youtube-dl version at this moment is **2016.03.06**. Read [our FAQ](https://github.com/rg3/youtube-dl/blob/master/README.md#how-do-i-update-youtube-dl) if you have troubles updating.
+- [ ] I've verified that I'm running youtube-dl version **2016.03.06**
+
+**What is the purpose of this issue? Put an `x` to the relevant box**
+- [ ] Site request (add support for a new website)
+- [ ] Feature request (request new functionality)
+- [ ] Bug report (encountered problems with youtube-dl)
+- [ ] Other, namely ...
+
+*If the purpose of this issue is a site request, bug report or you are unsure about its relevance please include a log as described below.*
+
+**Please include the full output of youtube-dl when run with `-v`**, i.e. **add** `-v` flag to **your command line**, copy the **whole** output and post it below wrapped in ``` for better formatting. It should look similar to this:
+```
+$ youtube-dl -v <your command line>
+[debug] System config: []
+[debug] User config: []
+[debug] Command-line args: [u'-v', u'http://www.youtube.com/watch?v=BaW_jenozKcj']
+[debug] Encodings: locale cp1251, fs mbcs, out cp866, pref cp1251
+[debug] youtube-dl version 2015.12.06
+[debug] Git HEAD: 135392e
+[debug] Python version 2.6.6 - Windows-2003Server-5.2.3790-SP2
+[debug] exe versions: ffmpeg N-75573-g1d0487f, ffprobe N-75573-g1d0487f, rtmpdump 2.4
+[debug] Proxy map: {}
+...
+```
+
+**Brief description of the problem/request**
+
+*I am having a problem with ... I have tried to do ... and ... I expected that ... would happen, but instead ... happened. Example: I tried to download a file but the site was not supported. Please add support for site xyz. Another example: I encountered a bug when downloading a video from xyz. I have tried to do a and b.*
+
+**Suggested solution or other information**
+
+*In case you have suggestions for a solution or any other relevant information you can write it here*
+
+**Example URL**
```

Comment:
This can also be worded like: If the purpose of this issues is a site support request please provide at least one example URL of a video or a playlist you are trying to download.